src/ui/controls/color-picker/hsb-picker.tsx

```diff
@@ -18,12 +19,23 @@ interface HSBPickerProps {
 }
 
 interface HSBPickerState {
+    wasPrevHidden: boolean;
+    hueCanvasRendered: boolean;
     activeHSB: HSB;
     activeChangeHandler: (color: string) => void;
     hueTouchStartHandler: (e: TouchEvent) => void;
     sbTouchStartHandler: (e: TouchEvent) => void;
 }
 
+const HSBPickerDefaults: HSBPickerState = {
```

Comment:
Nice decision to reuse the object for each re-rendering. But you know, naming conventions, it should start with lowercase, particularly the whole acronym should we lowercase in the beginning.